src/AccordionToggle.js

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { elementType } from 'prop-types-extra';
+import { useBootstrapPrefix } from './ThemeProvider';
+import AccordionContext from './AccordionContext';
+
+const propTypes = {
+  /**
+   * Set a custom element for this component.
+   */
+  as: elementType,
+
+  /** @default 'accordion-toggler' */
+  bsPrefix: PropTypes.string,
+  /**
+   * A key that corresponds to the collapse component that gets triggered
+   * when this has been clicked.
+   */
+  eventKey: PropTypes.string.isRequired,
+
+  /**
+   * A callback function to perform additional operations with this toggler
+   * gets triggered.
+   */
+  onClick: PropTypes.func,
+};
+
+const defaultProps = {
+  as: 'div',
+};
+
+const AccordionToggle = React.forwardRef(
+  (
+    { as: Component, bsPrefix, children, className, eventKey, ...props },
+    ref,
+  ) => {
+    bsPrefix = useBootstrapPrefix(bsPrefix, 'accordion-toggler');
```

Comment:
this isn't a bootstrap CSS class i don't think

Reply:
hmm, what bootstrap CSS class should I apply to this component? I wasn't sure which one would make the most sense for what it's trying to do.

Reply:
i'm not sure there is one – do you see one in the docs?

Reply:
Looks like there's only `accordion` that's specific to this component, judging by the accordion example on the bootstrap docs. I don't think I should apply any bootstrap CSS class for this component.

Reply:
yup – so this can just render the `<a>` or whatever